pkg/client/client.go

```diff
@@ -143,6 +143,7 @@ func (c *ConmonClient) CreateContainer(ctx context.Context, cfg *CreateContainer
 		if err := req.SetBundlePath(cfg.BundlePath); err != nil {
 			return err
 		}
+		req.SetTerminal(true)
```

Comment:
can we add a CreateContainerConfig field for this?

Reply:
Yes